tools/python/run_CIs_for_branch.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import json
+import os
+import subprocess
+import sys
+import typing
+
+from run_CIs_for_external_pr import get_pipeline_names
+from util.platform_helpers import is_windows
+
+
+def _parse_args():
+    parser = argparse.ArgumentParser(
+        os.path.basename(__file__),
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description="""Run the CIs used to validate PRs for the specified branch.
+
+        Requires the Azure CLI with DevOps extension to be installed.
+          Azure CLI: https://learn.microsoft.com/en-us/cli/azure/install-azure-cli
+          DevOps extension: https://github.com/Azure/azure-devops-cli-extension
+
+        Configuration:
+          Login:`az login`
+          Configure ORT repo as default:
+            `az devops configure --defaults organization=https://dev.azure.com/onnxruntime project=onnxruntime`
```

Comment:
Should we add an example of the command line to run to use that command?

Reply:
Sorry - which command do you mean? The `az` commands to configure or for the actual script? 

Latter could be something like 
`python .\tools\python\run_CIs_for_branch.py --exclude training --include Linux me/MyBranch`